test/saithrift/tests/sail2.py

```diff
@@ -1324,3 +1324,202 @@ def runTest(self):
             self.client.sai_thrift_remove_vlan_member(vlan_member3)
             self.client.sai_thrift_remove_vlan(vlan_oid)
 
+@group('l2')
+class L2TripleTaggedTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        print ""
+        switch_init(self.client)
+        pcp_list=[0,0,0]
+        cfi_list=[0,0,0]
+        tpid_list=[0x8100, 0x88a8, 0x8100]
```

Comment:
Is it intentional that 8100 repeated again in the list?

Reply:
Yes Rita - it is as per the test doc, [test case 9](https://github.com/opencomputeproject/SAI/blob/master/test/saithrift/tests/SAITest.md)